jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -313,7 +313,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *object_p, /**< the ob
   JERRY_ASSERT (property_desc_p->is_writable_defined || !property_desc_p->is_writable);
 
   /* 1. */
-  ecma_extended_property_ref_t ext_property_ref;
+  ecma_extended_property_ref_t ext_property_ref = {0,0};
```

Comment:
Somehow, my push caused Travis to report that this variable could be used before being initialized, so I just added the obvious initialization...